lib/puppet.rb

```diff
@@ -36,6 +37,9 @@ module Puppet
   require 'puppet/environments'
 
   class << self
+    GettextSetup.initialize(File.absolute_path('../locales', File.dirname(__FILE__)))
```

Comment:
I'm ignorant of the scoping implications here. Should I move this outside the Puppet module definition?